frontend_egui/src/app.rs

```diff
@@ -389,7 +393,23 @@ impl SnowGui {
                     ui.close_menu();
                 }
                 ui.separator();
+                ui.menu_button("Recent workspaces", |ui| {
+                    for (i, path, display_name) in self.settings.get_recent_workspaces_for_display() {
```

Comment:
You have to call `ui.set_min_width(Self::SUBMENU_WIDTH);` here, otherwise egui may make the menu very short and wrap the text